src/app/landing/LandingPage.jsx

```diff
@@ -99,10 +109,15 @@ class LandingPage extends Component {
             </div>
 
             <picture>
+              <source
+                className="OnboardingImage"
+                media="(min-aspect-ratio: 2/3)"
+                srcSet="img/landing_page/step3.png"
+              />
               <img
                 className="OnboardingImage"
-                src="img/landing_page/step3.png"
-                alt=""
+                src="img/landing_page/step3_mobile.png"
+                alt="map showing food, clothing, and personal care markers"
```

Comment:
Wait, more importantly... step **1** is the map, isn't it? This is just a picture of a person.
(I think it might still be a good idea to still describe what it shows, though `alt=""` is acceptable for purely decorative pictures. Whatever you think.)

Reply:
oops, fixed